Sources/SwiftKuery/QueryResult.swift

```diff
@@ -92,6 +92,7 @@ public enum QueryResult {
                 }
                 return dictionary
             }
+            resultSet.done()
```

Comment:
Add comment to function to be clear the result set is consumed and can be closed.